### ISSUE TITLE
Command to add remote only if it doesn't exist

### ISSUE THREAD

**Issue by Lyberta** [conan-io/conan#1995]
Currently, there are 2 commands: `conan remote add` that adds a remote if it doesn't exist and returns error code if it does and `conan remote update` which updates remote if it exists and returns error code if it doesn't.

Sometimes, I have a complex script that contains several independent parts that both try to add the same remote. There is no way to do this cleanly without complex parsing of `conan remote list`. I think the best fix is to add a flag to `conan remote add` so it skips adding remote if it exists.

**Comment by memsharded** [comment]
Yes, the ``upsert`` functionality is missing, can be added. I think it might not skip, but maybe also to do ordering if the ``--insert`` argument is present.

**Comment by wagdav** [comment]
Hello,

I think this feature is already implemented and the ticket can be closed. 

```
$ conan --version
Conan version 1.6.0
$ conan remote add --help
[...]
  -f, --force           Force addition, will update if existing
```
I think that the `conan remote add --force` has exactly the described effect. Or am I missing something?

**Comment by Lyberta** [comment]
The spirit of this issue is not to change remote if it exists. `--force` will forcefully overwrite it.

**Comment by Minimonium** [comment]
@memsharded @Lyberta 
I have a few questions about that feature:
1. `conan remote list` consists of two keys, a remote name and an URL, what should be the behaviour of `--upsert` if the name exists, but an URL is not the same? Same as update?
2. I suppose there should be an error if one gonna try to `--upsert` with existing URL, but with different name?
3. Should the same option be implemented for `list_ref`?

**Comment by Lyberta** [comment]
Doing nothing when remote exists but has different URL is useful. Consider CI system where there are some default predefined URLs. Individual libraries could overwrite URLs and CI script that in my case checked for existence of Conan after building the library would keep the URL... Actually, my use case seems to be different. I think we can close this issue.
